ansible_ai_connect/ai/tests/test_apps.py

```diff
@@ -323,3 +243,37 @@ def test_reports_postman_rogue_setting(self):
             with self.assertRaises(KeyError):
                 app_config.get_reports_postman()
             self.assertInLog("Unexpected ANSIBLE_AI_ONE_CLICK_REPORTS_POSTMAN value: banana", log)
+
+    @override_settings(ENABLE_ARI_POSTPROCESS=False)
```

Comment:
You can drop the line above. The envvar should not do anything now.

Reply:
done